src/neuroconv/tools/nwb_helpers/_configure_backend.py

```diff
@@ -85,16 +25,16 @@ def configure_backend(
         The configuration model to use when configuring the datasets for this backend.
     """
     is_ndx_events_installed = is_package_installed(package_name="ndx_events")
-    ndx_events = sys.modules.get("ndx_events", None)
+    ndx_events = importlib.import_module("ndx_events") if is_ndx_events_installed else None
```

Comment:
This was giving me trouble somehow...

Reply:
Might have to do with the change a while back to use `find_spec` in `is_package_installed` instead of attempting to import the module, which would have added it to the `sys.modules`

Reply:
I see.